src/service/http/interceptors.ts

```diff
@@ -20,6 +20,11 @@ const updateNetworkError = (dispatch: AppDispatch, occurError: boolean) => {
 export const initAxiosInterceptors = (dispatch: AppDispatch, history: History) => {
   axiosIns.interceptors.request.use(
     config => {
+      if (config.method === 'get') {
+        config.data = {
+          unused: 0,
```

Comment:
What does this configuration do?

Reply:
Fix the bug of axios whose version is 0.19.0 or later

Reply:
https://segmentfault.com/q/1010000008943315

Reply:
ok, got it. Actually, we will not set charset, our content-type is a fixed value `application/vnd.api+json`

Reply:
The content-type configuration is invalid in v0.19.0 or later version and it has nothing to do with the setting content.